### DISCUSSION TITLE
v0.11.4

### DISCUSSION THREAD

**Discussion by AsamK** [AsamK/signal-cli#1058] [Releases]
**Full Changelog**: https://github.com/AsamK/signal-cli/compare/v0.11.3...v0.11.4

<hr /><em>This discussion was created from the release <a href='https://github.com/AsamK/signal-cli/releases/tag/v0.11.4'>v0.11.4</a>.</em>

**Comment by cuu508** [comment]
In the release assets there are:

* signal-cli-0.11.4-Linux-native.tar.gz
* signal-cli-0.11.4-Linux.tar.gz 

What is the difference between these two? Thanks!

**Reply by AsamK** [reply]
The native one is compiled with graalvm native image. It provides better performance but isn't guaranteed to work in all scenarios, due to issues with reflection.
